File: challenges/uri/1013.c
#include <stdio.h>

int abs(int n) {
    if (n < 0) {
        return n*-1;
    }
    return n;
}

int get_maior(int x, int y) {
    return (x + y + abs(x - y)) / 2;
}

int main() {
    int A, B, C;
    scanf("%i %i %i", &A, &B, &C);
    printf("%i eh o maior\n", get_maior(get_maior(A, B), C));
}

File: CHALLENGES/uri/1013.go
package main

import (
	"fmt"
)

func main() {
	var A, B, C int
	fmt.Scanf("%d %d %d", &A, &B, &C)
	fmt.Printf("%d eh o maior\n", maior(maior(A, B), C))
}

func maior(a, b int) int {
	return (a + b + abs(a-b)) / 2
}

func abs(a int) int {
	if a <= 0 {
		return a * -1
	}
	return a
}
